indico_jacow/client/MultipleAffiliationsSelector.jsx

```diff
@@ -127,42 +133,79 @@ const MultipleAffiliationsField = ({onChange, value, currentAffiliations}) => {
   };
 
   return (
-    <DndProvider backend={HTML5Backend}>
-      <Segment attached="top">
-        <SortableWrapper accept="affiliation">
-          <List divided relaxed>
-            {value.length > 0 ? (
-              value.map((affiliation, idx) => (
-                <DraggableAffiliation
-                  key={affiliation.id}
-                  index={idx}
-                  onMove={moveItem}
-                  affiliation={affiliation}
-                  onDelete={() => onChange(value.filter((_, i) => i !== idx))}
+    <>
+      <DndProvider backend={HTML5Backend}>
+        <Segment attached="top">
+          <SortableWrapper accept="affiliation">
+            <List divided relaxed>
+              {value.length > 0 ? (
+                value.map((affiliation, idx) => (
+                  <DraggableAffiliation
+                    key={affiliation.id}
+                    index={idx}
+                    onMove={moveItem}
+                    affiliation={affiliation}
+                    onDelete={() => onChange(value.filter((_, i) => i !== idx))}
+                  />
+                ))
+              ) : (
+                <Translate>There are no affiliations</Translate>
+              )}
+            </List>
+          </SortableWrapper>
+        </Segment>
+        <Segment attached="bottom">
+          <Dropdown
+            icon="search"
+            placeholder={Translate.string('Add affiliations...')}
+            options={affiliationOptions}
+            onSearchChange={searchChange}
+            onChange={addItem}
+            onBlur={() => setSearchResults([])}
+            selectOnBlur={false}
+            selectOnNavigation={false}
+            search={options => options}
+            selection
+            fluid
+          />
+        </Segment>
+      </DndProvider>
+      {searchUsed && (
+        <Message info>
+          <Translate>
+            If your affiliation is not listed, please check the{' '}
+            <Param name="ror" wrapper={<a href="https://ror.org/" />}>
+              ROR registry
+            </Param>
+            . If your affiliation can be found there but not here on Indico, contact{' '}
```

Comment:
I think this whole message is a bit too long/verbose right now...

Should I remove the part about checking if it's on ROR and requesting us to update with the latest ROR data altogether?

I think considering indico/indico-plugin-jacow#20 and the latest comments in there it would actually scale better if we did not direct people towards ROR at all, but rather every few month / after a major conference's CfA ended had a look at the newly created affiliations and then sent a list of those that sound ROR-worthy to them.

Reply:
That makes sense. I think even with the comment, people are unlikely to create the request themselves if they can just add it in Indico anyway.

Reply:
I agree, not redirecting users to RoR might be better. I also think that with this PR users are more likely to search and then add. The only concern I would have is people not intuitively selecting the correct affiliation since it might have a different name in the drop-down results and we do not have the "Other names" descriptor that can help the user make sense of the results like in the RoR repository

Reply:
That's why I recommend people to enter acronym versions of the name etc. as alternative names when creating a new affiliation ;)

Reply:
![image](https://github.com/user-attachments/assets/b0c8f6de-36a8-4876-927e-95f82cb02eca)

what do you think?

Reply:
Could be even shorter, something like `Affiliation not found in the ROR registry. Add it manually`, but this is also good.